core/test/RectGrid_test.cpp

```diff
@@ -18,10 +23,16 @@
 #include <fstream>
 
 const std::string filename = "RectGrid_test.nc";
+const std::string partition_filename = "partition_metadata_1.nc";
```

Comment:
Add a TODO (or FIXME?) to ensure that this netCDF file gets removed and generated when the issue #379 changes are merged.

Reply:
I will address this as part of #455 .